services/api/src/app.py

```diff
@@ -130,26 +133,28 @@ async def request(
     return response
 
 
-async def _blocking_response(response: ResponseModel):
-    logger.info(f"Responding to SID: `{response.session_id}`:")
-    response.log(logger)
+@sm.on("connect")
+async def connect(session_id: str, environ: Dict):
+    params = environ.get("QUERY_STRING")
```

Comment:
Where is this environment variable defined? I don't see any changes to the compose file / .env file